plugins/ua_network_tcp.c

```diff
@@ -291,12 +291,6 @@ ServerNetworkLayerTCP_closeConnection(UA_Connection *connection) {
         return;
     connection->state = UA_CONNECTION_CLOSED;
 #endif
-#if UA_LOGLEVEL <= 300
```

Comment:
yes, but you can just use it for the recent case (in the non-modified) master branch